js/Round.js

```diff
@@ -1,30 +1,46 @@
 class Round {
-  constructor(categories, dailyDoubleCount) {
+  constructor(categories, dDCount) {
     this.categories = categories;
     this.clues = [];
-    this.dailyDoubleCount = dailyDoubleCount;
-    // this.dailyDoubleLocations = [[i, j], [i2, j2]];
+    this.dDCount = dDCount;
+    this.dDLocations = [];
   }
 
   randomizeDailyDoubles() {
-    let col = Math.floor(Math.random() * 4)
-    let row = Math.floor(Math.random() * 4)
-    this.dailyDoubleLocations.push([col, row])
+    for (let i = 0; i < this.dDCount; i++) {
+      const index = Math.floor(Math.random() * this.clues.length);
+      this.dDLocations.push(index);
+    }
+    if (this.dDCount === 2 && this.dDLocations[0] === this.dDLocations[1]) {
```

Comment:
This checks for the 1 in 256 chance that two daily doubles in the second round are in the same spot and generates new random numbers until they are different.

Reply:
So this essentially  says if it happens to land on the same spot, change dD  location?